packages/suite/src/components/suite/Menu/components/TopMenu/index.tsx

```diff
@@ -67,23 +70,35 @@ interface Props {
 
 const TopMenu = (props: Props) => {
     const { deviceCount } = props;
-    const [previousDeviceCount, setPreviousDeviceCount] = useState(deviceCount);
+    const [localCount, setLocalCount] = useState<number | null>(null);
     const [triggerAnim, setTriggerAnim] = useState(false);
 
+    const countChanged = localCount && localCount !== deviceCount;
+    const timerRef = useRef<number | undefined>(undefined);
```

Comment:
use timeout reference as number

Reply:
looks like It needs to be done using `useRef`
When using number var eslint complains: 
> Assignments to the 'timerRef' variable from inside React Hook useEffect will be lost after each render. To preserve the value over time, store it in a useRef Hook and keep the mutable value in the '.current' property. Otherwise, you can move this variable directly inside useEffect.